testnet/stacks-node/src/tests/signer.rs

```diff
@@ -7,6 +7,7 @@ use std::time::{Duration, Instant};
 use std::{env, thread};
 
 use clarity::vm::types::QualifiedContractIdentifier;
+use hashbrown::HashMap as hash_map;
```

Comment:
unused

Reply:
I use this to pass a hashbrown hashmap to calculate_coordinator